packages/js-dpp/lib/errors/consensus/basic/dataContract/DataContractImmutablePropertiesUpdateError.js

```diff
@@ -0,0 +1,39 @@
+const AbstractBasicError = require('../AbstractBasicError');
+
+class DataContractImmutablePropertiesUpdateError extends AbstractBasicError {
+  /**
+   * @param {BaseOperation[]} diff
```

Comment:
We can't pass instances of classes to arguments, since they encoded with cbor to pass to clients. Please use only plain data.